electrum/plugins/bitbox02/bitbox02.py

```diff
@@ -85,7 +85,16 @@ def has_usable_connection_with_device(self) -> bool:
             return False
         return True
 
-    def pairing_dialog(self, wizard: bool = True):
+    def get_soft_device_id(self) -> Optional[str]:
+        if self.handler is None:
+            # Can't do the pairing without the handler. This happens at wallet creation time.
+            return None
```

Comment:
@SomberNight 

When setting up the wallet in the wizard, this function is called while `handler` is still None. Not sure how to handle this, but just not returning any device id at this stage seems to work fine. Is this okay or should this always return an ID?

If you want, you can test the multisig with one device by enabling the optional passphrase and just choosing two different passphrases.

Reply:
Looks like the handler is None only when the devices are listed in the wizard. In our testing, this PR works as expected.